embuilder.py

```diff
@@ -280,6 +281,9 @@ def main():
   if auto_tasks:
     print('Building targets: %s' % ' '.join(tasks))
 
+  if USE_NINJA:
+    os.environ['EMBUILDER_PORT_BUILD_DEFERRED'] = '1'
```

Comment:
So its not currently possible use ninja in non-deferred mode?   I guess thats OK?

Reply:
Ninja still works in non-deferred mode. If you do `EMCC_USE_NINJA=1 test/runner.py core0` it will build the libraries on-demand in non-deferred mode with ninja. Deferred mode only makes sense when using embuilder though.